README.md

```diff
@@ -52,6 +58,33 @@ See `test_*.cmd` scripts for usage samples.
 
 ## Development
 
+### Linting
```

Comment:
thanks for the instructions.. maybe move this to the `shave-that-yak` branch so we don't couple it with this new `timestamp`-based proposal. 🤔 this way whether the proposal lives or dies, the linting comments live-on!

Reply:
done!